tekton/publish-run.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1alpha1
```

Comment:
What is this file used for? Isn't the release `Task` used within the release `Pipeline`?

Reply:
(I'm leaving it in there though I'm not opposed to taking it out as well :man_shrugging: )